packages/kbn-optimizer/src/optimizer/optimizer_config.test.ts

```diff
@@ -457,7 +457,7 @@ describe('OptimizerConfig::create()', () => {
           [Window],
         ],
         "invocationCallOrder": Array [
-          22,
+          25,
```

Comment:
__note:__ `fast-glob` uses `os.cpus()` method mocked in this test, that how we end up with the increased call order.